.github/workflows/go.yml

```diff
@@ -26,7 +26,7 @@ jobs:
         run: |
           docker pull jboss/keycloak:latest
           docker run -d -e KEYCLOAK_USER=admin -e KEYCLOAK_PASSWORD=secret -e KEYCLOAK_IMPORT=/tmp/gocloak-realm.json -v "`pwd`/testdata/gocloak-realm.json:/tmp/gocloak-realm.json" -p 8080:8080 --name keycloak jboss/keycloak:latest -Dkeycloak.profile.feature.upload_scripts=enabled
-          sleep 10
+          sleep 30
```

Comment:
The unit tests fail not because of this sleep is too small, 10 second is totally enough. The tests fail because of a Realm's cache. So I'm just restarting the failed pipeline.

Reply:
back to 10s